test/e2e/common/node/container_probe.go

```diff
@@ -216,7 +216,7 @@ var _ = SIGDescribe("Probing container", func() {
 		Testname: Pod liveness probe, container exec timeout, restart
 		Description: A Pod is created with liveness probe with a Exec action on the Pod. If the liveness probe call does not return within the timeout specified, liveness probe MUST restart the Pod.
 	*/
-	ginkgo.It("should be restarted with an exec liveness probe with timeout [MinimumKubeletVersion:1.20] [NodeConformance]", func() {
+	ginkgo.It("should be restarted with an exec liveness probe with timeout [MinimumKubeletVersion:1.20] [NodeConformance] [Feature:ExecProbeTimeout]", func() {
```

Comment:
we don't want to lose coverage of this functionality in OSS Nodeconformance test grid. Adding `Feature` will exclude this test. I sent a proposal to address it: https://github.com/kubernetes/enhancements/pull/3042

Also, this is a known issue that `e2eskipper.SkipUnlessFeatureGateEnabled` is not working as expected all the time.

I don't think we can take this PR as is. Perhaps looking into why skipper is not working as expected would be a better course of actions.

/hold

Reply:
According to: https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/

The ExecProbeTimeout feature is enabled in GA since 1.20.

Also, in the same doc:
"
A General Availability (GA) feature is also referred to as a stable feature. It means:
 - The feature is always enabled; you cannot disable it.
"

In other words, the `e2eskipper.SkipUnlessFeatureGateEnabled` skip would never apply - and the test would always run.

Reply:
Saying that - adding `[Feature:ExecProbeTimeout]` tag is not correct anymore, since the feature is in GA already.

Sergey, do you have any suggestions, how to follow-up with this issues?

Reply:
If the feature is always enabled, you can close this PR